pkg/operator/deploy/install.go

```diff
@@ -70,15 +70,15 @@ func (deploy *Deployer) installMeteringConfig() error {
 	if apierrors.IsNotFound(err) {
 		_, err = deploy.meteringClient.MeteringConfigs(deploy.config.Namespace).Create(deploy.config.MeteringConfig)
 		if err != nil {
-			return fmt.Errorf("Failed to create the MeteringConfig resource: %v", err)
+			return err
```

Comment:
Did we want an error message here?

Reply:
This belonged to a cherry-picked commit from master - this just reduced redundant error messages.